packages/plugin-persistent-cache/README.md

```diff
@@ -0,0 +1,18 @@
+# @vitejs/plugin-persistent-cache [![npm](https://img.shields.io/npm/v/@vitejs/plugin-persistent-cache.svg)](https://npmjs.com/package/@vitejs/plugin-persistent-cache)
+
```

Comment:
this could use some more docs:
- what exactly is being cached?
- how is invalidation handled?
- i'm assuming it applies to the results of other plugins as well?
- does it apply in both dev and prod?
- can you control what directory the info is persisted in?
- is there anything you need to do to use this on GitHub Actions?